cli/src/pcluster/validators/cluster_validators.py

```diff
@@ -1470,3 +1470,22 @@ def _validate(self, queues):
                     f"public IPs can only be assigned to instances launched with a single network interface.",
                     FailureLevel.ERROR,
                 )
+
+
+class PoolsValidator(Validator):
```

Comment:
This can be verified at Schema level using the [marshmallow Length validator](https://marshmallow.readthedocs.io/en/stable/marshmallow.validate.html#marshmallow.validate.Length)

Reply:
Done. I have moved it to Schema level.